test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/MSBuildTests.Test.cs

```diff
@@ -9,7 +9,10 @@ public class MSBuildTests_Test : AcceptanceTestBase<NopAssetFixture>
     private const string AssetName = "MSBuildTests";
 
     public static string? FormatBuildMatrixEntry(MethodInfo method, object?[]? data)
-        => $"{data![0]},{(Equals(TargetFrameworks.All.ToMSBuildTargetFrameworks(), data[1]) ? "multitfm" : data[1])},{data[2]},{((bool)data[3]! ? "Succeeded" : "Failed")}";
```

Comment:
As pointed out by @Youssef1313, this might be related to a fix that was recently done for the support of tuples. We need to investigate if that's a proble.=m.